CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 -   :bug: Make `kedro-mlflow` hook log parameters when the project is configured with the `OmegaConfigLoader` instead of raising an error ([#430](https://github.com/Galileo-Galilei/kedro-mlflow/issues/430))
 
+-   :bug: Fix Error when using kedro 0.18.1 with TemplatedConfigLoader ([#452](https://github.com/Galileo-Galilei/kedro-mlflow/issues/452))
```

Comment:
2 minor comments: 
- Can you please move it to the "unrealeased" section and create a "fixed" section under this one? 
-  Change the message to be slightly more explicit:  
":bug: Avoid error when using kedro==0.18.1 with ``TemplatedConfigLoader`` and no ``mlflow.yml`` configuration file ([#452](https://github.com/Galileo-Galilei/kedro-mlflow/issues/452))"


The final result should look like this: 

```markdown
# Changelog

## [Unreleased]

### Fixed
-   :bug: Avoid error when using kedro==0.18.1 with ``TemplatedConfigLoader`` and no ``mlflow.yml`` configuration file ([#452](https://github.com/Galileo-Galilei/kedro-mlflow/issues/452))

## [0.11.9] - 2023-07-23

```

Reply:
done